using-timescaledb/reading-data.md

```diff
@@ -282,31 +277,29 @@ This query will then output data in the following form:
  2017-09-22 |   9855
 ```
 Unlike with date_trunc, we can use a custom time interval with the time_bucket function,
-but in order to do gap filling properly with it we will also need to use time_bucket on our generated series.
+but in order for the generates times to align with the subselect we need use time_bucket on
+the first argument to generate_series.
 For example, let's say you want 1080 data points in the last two weeks, and as many graphing
```

Comment:
Not part of this PR, but comma should go from in front of "and" to after, I believe.